transactions/transaction_services.py

```diff
@@ -224,15 +225,32 @@ def save_card(cls, verify_resp):
         if verify_resp['data']['status'] == 'successful' \
                 and int(save_card) == 1:
             try:
-                user = User.active_objects.filter(email=email)
-                user.update(card_info=card_info)
-                message = '. Card details have been saved.'
-            except:  # noqa
+                user = User.active_objects.filter(email=email).first()
+                user_count = CardInfo.active_objects.all_objects().filter(user=user).count()
```

Comment:
**card_count** would be a more suitable variable name here

Reply:
@Victoradukwu,  I fixed this.